client/executor/src/wasm_runtime.rs

```diff
@@ -149,12 +179,12 @@ const MAX_RUNTIMES: usize = 2;
 /// the memory reset to the initial memory. So, one runtime instance is reused for every fetch
 /// request.
 ///
-/// The size of cache is equal to `MAX_RUNTIMES`.
+/// The size of cache is configurable via the cli option `--runtime-cache-size`.
 pub struct RuntimeCache {
 	/// A cache of runtimes along with metadata.
 	///
 	/// Runtimes sorted by recent usage. The most recently used is at the front.
-	runtimes: Mutex<[Option<Arc<VersionedRuntime>>; MAX_RUNTIMES]>,
+	runtimes: Mutex<LruCache<VersionedRuntimeId, VersionedRuntimeValue>>,
```

Comment:
```suggestion
	runtimes: Mutex<LruCache<VersionedRuntimeId, VersionedRuntime>>,
```

Why don't we just use `VersionedRuntime` here, as before?

Reply:
To avoid storing twice the meta-data already present in VersionedRuntimeId

Reply:
Can you not remove it from the VersionedRuntime? The duplicate data? Or at least some of it? Nevertheless, nothing of that is really big, so I see no problem. Please also do this change.

Reply:
Done